src/platform_cfg.cpp

```diff
@@ -296,6 +297,10 @@ void operator >> (const YAML::Node& node, CMacYamlInfo & mac_info) {
     if (! utl_yaml_read_uint16(node, "vlan", mac_info.m_vlan, 0, 0xfff)) {
         mac_info.m_vlan = 0;
     }
+
+    if (node.FindValue("tunnel")) {
+        mac_info.m_tunnel = Tunnel::Parse(node["tunnel"]);
+    }
```

Comment:
```
src/platform_cfg.cpp.1.o: In function `operator>>(YAML::Node const&, CMacYamlInfo&)':
/opt/trex/linux/build/../../src/platform_cfg.cpp:302: undefined reference to `Tunnel::Parse(YAML::Node const&)'
collect2: error: ld returned 1 exit status
```
The change leads to build errors

Reply:
Note this only happens when building `linux` and not `linux_dpdk`

Reply:
The GTP-U encap/decap patch depends on DPDK, it utilizes a DPDK TX/RX callback on the interface. Hence, the PR cannot be compiled without DPDK.